test/optimization/test_converters.py

```diff
@@ -53,14 +54,19 @@ class TestConverters(QiskitOptimizationTestCase):
 
     def test_empty_problem(self):
         """ Test empty problem """
-        op = QuadraticProgram()
-        conv = InequalityToEquality()
-        op = conv.convert(op)
-        conv = IntegerToBinary()
-        op = conv.convert(op)
-        conv = LinearEqualityToPenalty()
-        op = conv.convert(op)
-        _, shift = op.to_ising()
+        try:
+            warnings.filterwarnings(action="ignore", category=DeprecationWarning)
```

Comment:
I am not sure again why using these causes deprecation warnings. It looks like valid use to me - did the deprecation come from somehow the old to ising converters being pulled in which case maybe something should be done about that since valid use should not cause warnings. Again maybe an underlying library?